test/models/catagory_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```

Comment:
I am not able to see: 
1. Unit tests for all the most important components of the RoR application.
2. Integration tests for all the most important components of the RoR application.
Please add tests, this is important since it discovers defects/bugs before the delivery to the client, which guarantees the quality of the software.